src/openforms/appointments/contrib/qmatic/tests/test_plugin.py

```diff
@@ -355,15 +356,16 @@ def test_get_times_server_error(self, m, status_code):
     @requests_mock.Mocker()
     @given(st.integers(min_value=400, max_value=499))
     def test_get_times_client_error(self, m, status_code):
-        m.get(requests_mock.ANY, status_code=status_code)
-        product = AppointmentProduct(identifier="k@pu77", name="Kaputt")
-        location = AppointmentLocation(identifier="1", name="Bahamas")
+        with c_profile():
```

Comment:
I can't get this test to fail again now, but I'd like to leave the profiling code in for the next time it fails so I have data to look at.